helper-scripts/compile-geth.sh

```diff
@@ -77,7 +77,7 @@ fi
 
 if [[ "$e" = "docker" ]] ; then
     echo -e "${GREEN}Building geth using docker!${NC}"
-    docker build -t vulcanize/go-ethereum -f Dockerfile .
+    docker build --platform linux/arm64 -t vulcanize/go-ethereum -f Dockerfile .
```

Comment:
I think we should remove the platform flag for the time being. When we add platform-specific support we can have various conditions based on user setup.